source/fundamentals/connection.txt

```diff
@@ -245,6 +245,16 @@ that Context deadline for the operation. If the context does not have a
 deadline, the driver derives a new Context from the given Context using
 the ``Timeout`` option set on the ``Client``.
 
+.. note:: Retryability
+
+   This version of the {+driver-short+} enables retryable reads and
+   writes by default. If you set a ``Timeout`` option on your ``Client``
+   and your operation requires a retry, the driver retries the operation
+   as many times as possible before the timeout expires. Once the timeout
+   expires, the driver returns a timeout error. See the Server manual
+   for more information about :manual:`retryable reads </core/retryable-reads/>` and
```

Comment:
Suggestion:
Since the snooty file imports the refs from the Server Manual, you can use the retryable-reads and retryable-writes ref targets instead of the manual directive. These targets are hypothetically more durable than the url.